azure_monitor/examples/traces/request.py

```diff
@@ -6,19 +6,21 @@
 import requests
 from opentelemetry import trace
 from opentelemetry.ext import http_requests
-from opentelemetry.sdk.trace import TracerSource
+from opentelemetry.sdk.trace import TracerProvider
 from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
 
 from azure_monitor import AzureMonitorSpanExporter
 
-trace.set_preferred_tracer_source_implementation(lambda T: TracerSource())
+trace.set_preferred_tracer_provider_implementation(lambda T: TracerProvider())
```

Comment:
OT code for this is not published yet, do we want to merge this right now or until beta is released?

Reply:
Yeah I think that's why the build is failing. I will probably leave the logic as `tracer_source` until OT is published as to not have breaking code.

Reply:
EDIT: Actually I'll probably comment on the tests for now until the OT beta is released.